### DISCUSSION TITLE
2022-02-28 PyAerocom meeting

### DISCUSSION THREAD

**Discussion by avaldebe** [metno/pyaerocom#591] [Development meetings]
## Follow up from last meeting

Feb 21th #587

## Issues & Discussions

- #588
- #584
- #580
- #551
- #543
- #539

## PRs

- #590
- #589
- #582
- #550

## Short presentation

Jan will present what he learned from PR #550

## For next

- [ ]  Python packaging mini workshop with Álvaro, Krister, Ove, Daniel and Charlie. Sometime early March.
- [ ]  Intro to intake catalogues mini workshop, mid March?

**Comment by avaldebe** [comment]
## release schedule

- need to keep up with iris releases
- up to 4 releases a year: one minor and 3 patches
- at least 0.12.2 end of April and 0.13.0 end of October (?)

**Comment by jgriesfeller** [comment]
Here's the presentation:

- [pdf](https://github.com/metno/pyaerocom/files/8153194/Pyaerocom_.introducing.a.new.calculated.variable.pdf)
- [link to Google Drive](https://docs.google.com/presentation/d/1lah7TEMS7qKCIV5dOBcLc-MP4l4nPBMVveoDbK9LmbY/edit?usp=sharing)
